cmd/vinegar/vinegar.go

```diff
@@ -91,6 +92,13 @@ func main() {
 			if err := InstallWebview2(&pfx); err != nil {
 				log.Fatal(err)
 			}
+		case "reportinfo":
+			report, err := util.GenerateInfo(*configPath)
```

Comment:
Pass the configuration struct, not the file. As the struct member will just be the contents of the file.